rest/replicatortest/replicator_conflict_test.go

```diff
@@ -2119,6 +2119,168 @@ func TestActiveReplicatorConflictRemoveCVFromCache(t *testing.T) {
 	})
 }
 
+func TestActiveReplicatorV4DefaultResolverWithTombstoneLocal(t *testing.T) {
+	base.RequireNumTestBuckets(t, 2)
+
+	sgrRunner := rest.NewSGRTestRunner(t)
+	// v4 protocol only test
+	sgrRunner.RunSubprotocolV4(func(t *testing.T) {
+		rt1, rt2, remoteURLString := sgrRunner.SetupSGRPeers(t)
+		remoteURL, err := url.Parse(remoteURLString)
+		require.NoError(t, err)
+		ctx1 := rt1.Context()
+
+		docID := rest.SafeDocumentName(t, t.Name())
+		rt1Version := rt1.PutDoc(docID, `{"source":"rt1","channels":["alice"]}`)
+		// delete local version
+		rt1Version = rt1.DeleteDoc(docID, rt1Version)
+		rt1.WaitForPendingChanges()
+		// create conflicting update on rt2
+		rt2Version := rt2.PutDoc(docID, `{"source":"rt2","channels":["alice"]}`)
+		rt2Version = rt2.UpdateDoc(docID, rt2Version, `{"source":"rt2-updated","channels":["alice"]}`)
+		rt2.WaitForPendingChanges()
+
+		replicationID := rest.SafeDocumentName(t, t.Name())
+		resolverFunc, err := db.NewConflictResolverFuncForHLV(ctx1, db.ConflictResolverDefault, "", rt1.GetDatabase().Options.JavascriptTimeout)
+		require.NoError(t, err)
+		stats, err := base.SyncGatewayStats.NewDBStats(replicationID, false, false, false, nil, nil)
+		require.NoError(t, err)
+		replicationStats, err := stats.DBReplicatorStats(replicationID)
+		require.NoError(t, err)
+
+		ar, err := db.NewActiveReplicator(ctx1, &db.ActiveReplicatorConfig{
+			ID:          replicationID,
+			Direction:   db.ActiveReplicatorTypePushAndPull,
+			RemoteDBURL: remoteURL,
+			ActiveDB: &db.Database{
+				DatabaseContext: rt1.GetDatabase(),
+			},
+			ChangesBatchSize:           200,
+			ReplicationStatsMap:        replicationStats,
+			ConflictResolverFuncForHLV: resolverFunc,
+			CollectionsEnabled:         !rt1.GetDatabase().OnlyDefaultCollection(),
+			Continuous:                 true,
+			SupportedBLIPProtocols:     sgrRunner.SupportedSubprotocols,
+		})
+		require.NoError(t, err)
+		defer func() {
+			require.NoError(t, ar.Stop())
+		}()
```

Comment:
This is fine but I wonder you can just use `rt1.CreateReplication` to reduce boilerplate? If this isn't easy, I wonder if we should make it easier?

Additionally, I have preference for using passiveRT and activeRT rather than remember that rt1 and rt2 are active